mjs_firmware.ino

```diff
@@ -1,12 +1,10 @@
 /*******************************************************************************
    Copyright (c) 2016 Thomas Telkamp, Matthijs Kooijman, Bas Peschier, Harmen Zijp
-
    Permission is hereby granted, free of charge, to anyone
    obtaining a copy of this document and accompanying files,
    to do whatever they want with them without any restriction,
    including, but not limited to, copying, modification and redistribution.
    NO WARRANTY OF ANY KIND IS PROVIDED.
-
```

Comment:
This removes empty lines for no apparent reason, so I'd rather have this part of the commit removed.

Reply:
@matthijskooijman Certainly, this change is not intentional. I will revert it.
```suggestion


```